modmesh/onedim/linear_scalar.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2025, Ting-Yu Chuang <tychuang.cs10@nycu.edu.tw>
+# BSD 3-Clause License, see COPYING
```

Comment:
Please add the full license text in the file header.

Reply:
Got it.